README.md

```diff
@@ -85,6 +85,13 @@ $ ./example -ds 1337 foo bar
 Opt { debug: true, speed: 1337, input: "foo", output: Some("bar") }
 ```
 
+If you want to disable the color-terminal support (and other `clap` default-feautres) add `default-features = false` to the `structopt` dependency:
+```toml
+[dependencies]
+structopt = { version = "0.1.0", default-features = false }
+structopt-derive = "0.1.0"
+```
```

Comment:
If you want to disable all the `clap` features (color, suggestions...), add [...]

I would prefer this doc to be in src/lib.rs than in README.md.

After that, I'll merge it.

Thanks!